docs/deployment.md

```diff
@@ -0,0 +1,205 @@
+
+#### Auth
+
+| Variable                       | Default                                                          | Description                                                                |
+| ------------------------------ | ---------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `ACCESS_TOKEN_DELETION_DAYS`   | `30`                                                             | days until expired or revoked access tokens are deleted                    |
+| `ACCESS_TOKEN_EXPIRY_SECONDS`  | `10 * 60`                                                        | expiry time for access tokens (default: 10 minutes)                        |
+| `ADMIN_PORT`                   | `3003`                                                           | GraphQL Server port                                                        |
+| `AUTH_DATABASE_URL`            | `postgresql://postgres:password@localhost:5432/auth_development` | Postgres database URL                                                      |
+| `AUTH_SERVER_DOMAIN`           | `http://localhost:3006`                                          | endpoint of this Open Payments Auth Server                                 |
+| `COOKIE_KEY`                   | 32 random bytes                                                  | signed cookie key                                                          |
+| `DATABASE_CLEANUP_WORKERS`     | `1`                                                              | number of workers processing expired or revoked access tokens              |
+| `IDENTITY_SERVER_DOMAIN`       | `http://localhost:3030/mock-idp/`                                | endpoint of the identity server controlled by the Account Servicing Entity |
+| `IDENTITY_SERVER_SECRET`       | `replace-me`                                                     | API key                                                                    |
+| `INCOMING_PAYMENT_INTERACTION` | `false`                                                          | flag - incoming payments grants are interactive or not                     |
+| `INTROSPECTION_HTTPSIG`        | `false`                                                          | flag - check http signature on introspection requests                      |
+| `LOG_LEVEL`                    | `info`                                                           | [Pino Log Level](https://getpino.io/#/docs/api?id=levels)                  |
+| `NODE_ENV`                     | `development`                                                    | node environment, `development`, `test`, or `production`                   |
+| `PORT`                         | `3006`                                                           | port of this Open Payments Auth Server                                     |
```

Comment:
Maybe good to mention this should be the same as the port in `AUTH_SERVER_DOMAIN`?